src/main/telemetry/jetiexbus.c

```diff
@@ -102,26 +102,52 @@ const uint8_t exDataTypeLen[]={
     [EX_TYPE_GPS] = 4
 };
 
-typedef struct exBusSensor_s{
+typedef struct exBusSensor_s {
     const char *label;
     const char *unit;
-    int32_t value;
     const uint8_t exDataType;
     const uint8_t decimals;
+    uint8_t parameter;
 } exBusSensor_t;
 
 #define DECIMAL_MASK(decimals) (decimals << 5)
 
 // list of telemetry messages
-// after every 15 sensors a new header has to be inserted (e.g. "CF-Dev 1.12 S2")
+// after every 15 sensors a new header has to be inserted (e.g. "BF D2")
+
 exBusSensor_t jetiExSensors[] = {
-    { "CF-Dev 1.12 S1", "",     0,      0,             0 },                     // device descripton
-    { "Voltage",        "V",    0,      EX_TYPE_14b,   DECIMAL_MASK(1) },
-    { "Current",        "A",    0,      EX_TYPE_14b,   DECIMAL_MASK(2) },
-    { "Altitude",       "m",    0,      EX_TYPE_14b,   DECIMAL_MASK(1) },
-    { "Capacity",       "mAh",  0,      EX_TYPE_22b,   DECIMAL_MASK(0) },
-    { "frames lost",    " ",    0,      EX_TYPE_22b,   DECIMAL_MASK(0) },       // for debug only
-    { "time Diff",      "us",   0,      EX_TYPE_14b,   DECIMAL_MASK(0) }        // for debug only
+    { "BF D1",       "",      EX_TYPE_DES,   EX_TYPE_NONE,    EX_TYPE_SENSOR_ENABLED},     // device descripton
+    { "Voltage",     "V",     EX_TYPE_14b,   DECIMAL_MASK(1), EX_TYPE_SENSOR_DISABLED},
+    { "Current",     "A",     EX_TYPE_14b,   DECIMAL_MASK(2), EX_TYPE_SENSOR_DISABLED},
+    { "Altitude",    "m",     EX_TYPE_14b,   DECIMAL_MASK(2), EX_TYPE_SENSOR_DISABLED},
+    { "Capacity",    "mAh",   EX_TYPE_22b,   DECIMAL_MASK(0), EX_TYPE_SENSOR_DISABLED},
+    { "Power",       "W",     EX_TYPE_22b,   DECIMAL_MASK(1), EX_TYPE_SENSOR_DISABLED},
+    { "Roll angle",  "\xB0",  EX_TYPE_14b,   DECIMAL_MASK(1), EX_TYPE_SENSOR_DISABLED},
+    { "Pitch angle", "\xB0",  EX_TYPE_14b,   DECIMAL_MASK(1), EX_TYPE_SENSOR_DISABLED},
+    { "Heading",     "\xB0",  EX_TYPE_14b,   DECIMAL_MASK(1), EX_TYPE_SENSOR_DISABLED},
+    /*{ "free",        "\xB0",   EX_TYPE_14b,   DECIMAL_MASK(1), EX_TYPE_SENSOR_DISABLED},
```

Comment:
What's the purpose of this commented out section? Should this be removed?

Reply:
Is removed.